sponsorship/src/main/java/com/kyriba/conference/sponsorship/domain/EmailMessage.java

```diff
@@ -0,0 +1,47 @@
+package com.kyriba.conference.sponsorship.domain;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+public class EmailMessage
```

Comment:
Some params are required for this DTO, so corresponding constructor should be added. Remaining could be set using setters.